test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/EventSourceTests.cs

```diff
@@ -56,7 +56,7 @@ public EventSourceTests()
 
             var stop = Assert.Single(events, e => e.EventName == "ConnectionStop");
             Assert.All(new[] { "connectionId" }, p => Assert.Contains(p, stop.PayloadNames));
-            Assert.Same(KestrelEventSource.Log, stop.EventSource);
+            Assert.Same(KestrelEventSource.Log, stop?.EventSource);
```

Comment:
The `?` is unnecessary since `Assert.Single` will fail if it can't find the stop event.